doc/presentations.rst

```diff
@@ -24,21 +52,44 @@ specific subject areas:
 Videos
 ======
 
-- An introduction to scikit-learn `Part
-  I <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=107>`_ and
-  `Part II <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=111>`_ at Scipy 2013
-  by `Gael Varoquaux`_, `Jake Vanderplas`_  and `Olivier Grisel`_. Notebooks on
+- The `:probabl YouTube channel <https://www.youtube.com/@probabl_ai>`_
```

Comment:
I am a bit uneasy about adding a Probabl resource in general, and I am definitely uneasy because it is the first resource listed in the Videos and at the very minimum the official scikit-learn channel should be the first one listed.

This is a fine line to navigate and I would be more that happy to hear other opinions on this though ...

I agree that the [Whiteboard series](https://www.youtube.com/playlist?list=PLSIzlWDI17bS025ph6R0W_3RKM0qJ3qoO) is kind of relevant for scikit-learn and the videos I watched are good, but then we also open the door for PRs that want to use SEO optimization to have their scikit-learn content listed here and I am not very keen on opening this door ...

Reply:
Alright, let's keep the videos restricted to conferences and talks given by core maintainers.